Userland/Libraries/LibGUI/TextDocument.cpp

```diff
@@ -315,6 +315,7 @@ String TextDocument::text() const
 
 String TextDocument::text_in_range(const TextRange& a_range) const
 {
+    if (is_empty()) return String("");
```

Comment:
The commit message "Fix TextDocument crashing when the selected text is all text" doesn't really explain what this is fixing. Please improve it :)